cmd/kubernetes_create.go

```diff
@@ -175,6 +175,9 @@ var kubernetesCreateCmd = &cobra.Command{
 			for _, v := range strings.Split(removeapplications, ",") {
 				if utility.CheckAPPName(v) {
 					rmApp = append(rmApp, fmt.Sprintf("-%s", v))
+					if utility.Contains(rmApp, "-metrics-server") || utility.Contains(rmApp, "-traefik-v2-nodeport") {
+						continue
+					}
 				} else {
 					utility.Warning("the app that tries to remove %s is not valid", v)
 					os.Exit(1)
```

Comment:
You'll need to modify the installApplications field pretty much always too to explicitly add metrics server and traefik.